tools/harness-thci/OpenThread.py

```diff
@@ -195,6 +195,7 @@ class OpenThreadTHCI(object):
     IsBorderRouter = False
     IsBackboneRouter = False
     IsHost = False
+    isPowerDown = False
```

Comment:
Is it necessary? I think Harness would always call `setDefaultValues` to initialize.

Reply:
I had inspection warnings that the attribute is defined outside of __init__. I think it is good to initialize it.

Reply:
It's quite easy to forget resetting an attribute in `setDefaultValues`, which caused several bugs that are difficult to debug. 

So, I intentionally defined attributes only in `setDefaultValues` so that it fails immediately and loudly as long as any attribute is not properly initialized and reset in `setDefaultValues`. 

That should be enough benefit for us to bear the warnings?

Reply:
@tomaszkob89 , thoughts on this?

Reply:
I removed the line according to @simonlingoogle suggestion